confgenerator/confgenerator.go

```diff
@@ -64,7 +64,9 @@ func (uc *UnifiedConfig) GenerateOtelConfig(hostInfo *host.InfoStat) (string, er
 			Config: map[string]interface{}{
 				"user_agent": userAgent,
 				"metric": map[string]interface{}{
-					"prefix": "agent.googleapis.com/",
+					// Receivers are responsible for sending fully-qualified metric names.
```

Comment:
Can you open a ticket (P3) about writing a test to verify `custom.googleapis.com` metrics aren't slipping through?

Reply:
Done.